soc/arm64/nxp_imx/imx8m/Kconfig.soc

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+choice
+prompt "NXP i.MX8M A53 Selection"
+depends on SOC_SERIES_IMX8M_A53
+
+config SOC_IMX8MM_A53
+	bool "NXP i.MX8MM A53"
+	select ARM64
+	select CPU_CORTEX_A53
+	select ARM_ARCH_TIMER
+	select GIC_V3
+	select HAS_MCUX if CLOCK_CONTROL
```

Comment:
Why do these need to be conditional on `CLOCK_CONTROL`? If they were selected unconditionally, would there be negative affects?

Reply:
Currently NXP HAL for arm64 support is not ready, so MCUX is not supported. After MCUX HAL is ready for ARM64, this could be unconditional.